src/foremast/awslambda/api_gateway_event/api_gateway_event.py

```diff
@@ -112,12 +112,41 @@ def add_permission(self):
         statement_id = '{}_api_{}'.format(self.app_name, self.trigger_settings['api_name'])
         principal = 'apigateway.amazonaws.com'
         lambda_alias_arn = get_lambda_alias_arn(self.app_name, self.env, self.region)
+        lambda_arn = get_lambda_arn(self.app_name, self.env, self.region)
+        resource_name = self.trigger_settings.get('resource', '')
+        resource_name = resource_name.replace("/", "")
```

Comment:
Inconsistent quote usage. Please stick to single quotes like the line above this.